### PR TITLE
Comms for SSO Service Migration

### PR DESCRIPTION
**Pull request by patricksimonian**
- added a change log doc to serve as a single source of truth for the TIBs as well as devhub docs for SSO migration

### CODE REVIEW

File: docs/CHANGE_LOG/02.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  - dev and its stateful data
+  - test and its stateful data
+  - prod and its stateful data
+  - any internal testing instances as needed
+- the migration will occur in stages with respect to the SSO environments with the first migration being that of the dev instances followed by test and production in one week increments 
+- once the migration for an environment is complete it is __highly recommended__ that teams check their applications for normal operation and behaviour. We do not expect any issues to arise for applications beside losing login sessions. 
+
+## More Information
+- Approximate Time of Change: The __tentative__ dates for the migration are as follows:
+  - January 28th 2021: Dev migration to 4.5
```
Comment (ShellyXueHan) [inline docs/CHANGE_LOG/02.md:26]:
```suggestion
  - January 28th 2021: Dev migration to OCP4.5
```

File: docs/CHANGE_LOG/02.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  - February 4th 2021: Test migration to 4.5
```
Comment (ShellyXueHan) [inline docs/CHANGE_LOG/02.md:27]:
```suggestion
  - February 4th 2021: Test migration to OCP4.5
```

File: docs/CHANGE_LOG/02.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+  - February 11th 2021: Prod migration to 4.5
```
Comment (ShellyXueHan) [inline docs/CHANGE_LOG/02.md:28]:
```suggestion
  - February 11th 2021: Prod migration to OCP4.5
```

File: docs/CHANGE_LOG/02.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+- this includes:
+  - dev and its stateful data
+  - test and its stateful data
+  - prod and its stateful data
+  - any internal testing instances as needed
+- the migration will occur in stages with respect to the SSO environments with the first migration being that of the dev instances followed by test and production in one week increments 
+- once the migration for an environment is complete it is __highly recommended__ that teams check their applications for normal operation and behaviour. We do not expect any issues to arise for applications beside losing login sessions. 
+
+## More Information
+- Approximate Time of Change: The __tentative__ dates for the migration are as follows:
```
Comment (patricksimonian) [inline docs/CHANGE_LOG/02.md:25]:
```suggestion
- The __tentative__ dates for the migration are as follows:
```

File: docs/CHANGE_LOG/02.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+## Changes
+
+- all instances of the SSO service will be migrated to the Openshift 4.5 (Silver) cluster
+- this includes:
+  - dev and its stateful data
+  - test and its stateful data
+  - prod and its stateful data
+  - any internal testing instances as needed
+- the migration will occur in stages with respect to the SSO environments with the first migration being that of the dev instances followed by test and production in one week increments 
+- once the migration for an environment is complete it is __highly recommended__ that teams check their applications for normal operation and behaviour. We do not expect any issues to arise for applications beside losing login sessions.
```
Comment (patricksimonian) [inline docs/CHANGE_LOG/02.md:22]:
```suggestion
- once the migration for an environment is complete it is __highly recommended__ that teams check their applications for normal operation and behaviour. We do not expect any issues to arise for applications beside losing login sessions
```

**Review by ShellyXueHan — APPROVED** [review]
just to specify it's ocp4.5

File: docs/CHANGE_LOG/02.md [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+## More Information
+- The __tentative__ dates for the migration are as follows:
+  - January 28th 2021: Dev migration to 4.5
+  - February 4th 2021: Test migration to 4.5
+  - February 11th 2021: Prod migration to 4.5
+- Type of Change: scheduled maintenance
+- Service Bulletin No: in progress
+- Impact Notes:
+  - each migration will take place __outside of regular business hours__ between 6:30PM and 9:30PM.
+  - when a migration is progress the SSO service for the respective environment __will not be available during the maintenance window__ (approx. 3 hours)
```
Comment (patricksimonian) [inline docs/CHANGE_LOG/02.md:33]:
```suggestion
  - when a migration is in progress the SSO service for the respective environment __will not be available during the maintenance window__ (approx. 3 hours)
```

File: docs/CHANGE_LOG/02.md [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+## 2021-01-11
+
+The BC Gov's KeyCloak app that provides SSO service to product teams, is now fully prepared to be migrated off of the old Openshift 3.11 (Pathfinder) cluster and transfered to the new Openshift 4.5 (Silver) cluster. This migration effort has been a work in progress since early fall to ensure the smoothest possible transition between the clusters.  
+
+## Changes
+
+- all instances of the **KeyCloak SSO app** will be migrated to the Openshift 4.5 (Silver) cluster
+- this includes:
+  - dev instance and its stateful data
+  - test instanceand its stateful data
```
Comment (patricksimonian) [inline docs/CHANGE_LOG/02.md:18]:
```suggestion
  - test instance and its stateful data
```

**Pull request merged**